apps/studio/components/layouts/DatabaseLayout/DatabaseMenu.utils.tsx

```diff
@@ -64,30 +72,36 @@ export const generateDatabaseMenu = (
           url: `/project/${ref}/database/publications`,
           items: [],
         },
-        ...(enablePgReplicate
+        ...(showPgReplicate
           ? [
-              {
-                name: 'Replication',
-                key: 'replication',
-                url: `/project/${ref}/database/replication`,
-                items: [],
-              },
+              ...(enablePgReplicate
+                ? [
+                    {
+                      name: 'Replication',
+                      key: 'replication',
+                      url: `/project/${ref}/database/replication`,
+                      items: [],
+                    },
+                  ]
+                : [
+                    {
+                      name: 'Replication',
+                      key: 'replication',
+                      url: `/project/${ref}/database/replication`,
+                      label: 'Coming Soon',
+                      items: [],
+                    },
+                  ]),
```

Comment:
unrelated but can be simplied? difference between these 2 seems to just be the labels